protected/humhub/modules/user/views/layouts/main.php

```diff
@@ -11,7 +11,7 @@
 ?>
 <?php $this->beginPage() ?>
 <!DOCTYPE html>
-<html lang="en">
+<html lang="<?= Yii::$app->language ?>">
```

Comment:
I like the idea, although shouldn't this change on user language instead of default?

Reply:
@ArchBlood yes, as this layout can be used also for logged in users.
`Yii::$app->language` will have the value of the default language for guests.

Reply:
I think it's ok `Yii::$app->language` is always set to the actual user language.